tests/baselines/reference/api/typescript.d.ts

```diff
@@ -6433,7 +6433,18 @@ declare namespace ts {
         argumentIndex: number;
         argumentCount: number;
     }
+    enum CompletionInfoFlags {
+        None = 0,
+        MayIncludeAutoImports = 1,
+        IsImportStatementCompletion = 2,
+        IsContinuation = 4,
+        ResolvedModuleSpecifiers = 8,
+        ResolvedModuleSpecifiersBeyondLimit = 16,
+        MayIncludeMethodSnippets = 32
+    }
     interface CompletionInfo {
+        /** For performance telemetry. */
+        flags?: CompletionInfoFlags;
```

Comment:
We had discussed the possibility that this might subsume one or more existing flags, but it looks like that wasn't necessary?

Reply:
I think there is a property that we could delete in the future if we’re not worried about old clients using it.